tests/test_adcirc_mesh.py

```diff
@@ -168,3 +168,17 @@ def test_make_plot_flat_domain(nodes, elements, boundaries, mocker):
 
     mocker.patch('matplotlib.pyplot.show')
     mesh.make_plot()
+
+
+def test_mesh_equality(nodes, elements, boundaries, wet_nodes, wet_elements):
+    mesh_1 = AdcircMesh(nodes, elements, boundaries)
+    mesh_2 = AdcircMesh(nodes, elements, boundaries)
+    mesh_3 = mesh_1
+    mesh_4 = AdcircMesh(wet_nodes, wet_elements)
+
+    assert mesh_1 is not mesh_2
+    assert mesh_1 is mesh_3
```

Comment:
what's the difference between `is` and `==`?

Reply:
`is` checks if the two references refer to the same object (i.e. the literal same block in memory), whereas `==` will call `.__eq__()`